### PR TITLE
OCP: Added control resonse for NIST AU

### PR DESCRIPTION
**Pull request by Vincent056**
Added control response for AU-9(3) AU-5(1) AU-5(2) AU-12(1) AU-12(3) AU-3(2)

Add rule:

`alert_receiver_configured`
`cluster_logging_operator_exist`

### CODE REVIEW

File: applications/openshift/general/alert_receiver_configured/rule.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  in real time.
+
+  OpenShift provides multiple alert recivers integrations to send realtime alerts to different
+  services such as email, slack, pagerduty, webhooks, etc. [1][2]
+  
+  [1]https://docs.openshift.com/container-platform/latest/post_installation_configuration/configuring-alert-notifications.html#configuring-alert-receivers_configuring-alert-notifications
+  [2]https://docs.openshift.com/container-platform/latest/monitoring/managing-alerts.html#applying-custom-alertmanager-configuration_managing-alerts
+
+rationale: |-
+  Alerts are a critical part of the OpenShift Container Platform. Proper confiurating of
```
Comment (jhrozek) [inline applications/openshift/general/alert_receiver_configured/rule.yml:20]: typo: confiurating

File: applications/openshift/logging/cluster_logging_operator_exist/rule.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+title: Ensure that OpenShift Logging Operator is scanning the cluster
+
+description: |-
+  {{{ weblink(link="https://docs.openshift.com/container-platform/4.10/logging/cluster-logging.html",
+              text="The OpenShift Logging Operator") }}}
+  OpenShift Logging Operator provides ability to aggregate all the logs from the
+  OpenShift Container Platform cluster, such as node system audit logs, application
+  container logs, and infrastructure logs. OpenShift Logging aggregates these logs
+  from throughout OpenShift cluster and stores them in a default log store. [1]
```
Comment (jhrozek) [inline applications/openshift/logging/cluster_logging_operator_exist/rule.yml:11]: misplaced link?

File: applications/openshift/logging/cluster_logging_operator_exist/rule.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+rationale: |-
+  OpenShift Logging Operator is able to collect, aggregate, and manage logs.
```
Comment (jhrozek) [inline applications/openshift/logging/cluster_logging_operator_exist/rule.yml:14]: We already have a rule `audit_log_forwarding_enabled`. I'm not against this rule in principle, but is it even possible to have the log forwarding enabled but having this rule fail?

File: controls/nist_ocp4.yml [hunk 4]
```diff
@@ -3403,12 +3417,13 @@ controls:
...
 - id: AU-12(1)
-  status: pending
+  status: automated
   notes: |-
-    A control response is planned. Engineering progress can be tracked via:
-
-    https://issues.redhat.com/browse/CMP-181
-  rules: []
+    OpenShift synchronizes time by utilizing the time services of the underyling operating
+    operating system. This control is not applicable to OpenShift.
```
Comment (jhrozek) [inline controls/nist_ocp4.yml:3423]: Why do we say that the control is not applicable, but then list rules for it?

**Review by jhrozek — COMMENTED** [review]
Some nits and questions inline, overall looking good.

File: applications/openshift/logging/cluster_logging_operator_exist/rule.yml (hunk 3 above)
Comment (Vincent056) [reply applications/openshift/logging/cluster_logging_operator_exist/rule.yml:14]: I think it might be possible?

**Review by jhrozek — APPROVED** [review]
/lgtm

**Comment by jan-cerny** [comment]
@Vincent056 please rebase
@jhrozek feel free to merge once it's rebased

File: applications/openshift/logging/cluster_logging_operator_exist/tests/does_not_have_logging_instance.fail.sh [hunk 5]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
```
Comment (marcusburghardt) [inline applications/openshift/logging/cluster_logging_operator_exist/tests/does_not_have_logging_instance.fail.sh:2]: Nit: this empty line could be removed.

File: applications/openshift/logging/cluster_logging_operator_exist/tests/have_cluster_logging_instance.pass.sh [hunk 6]
```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
```
Comment (marcusburghardt) [inline applications/openshift/logging/cluster_logging_operator_exist/tests/have_cluster_logging_instance.pass.sh:2]: Nit: this empty line could be removed.

**Comment by jhrozek** [comment]
e2e:
```
    helpers.go:1028: Command '/go/src/github.com/ComplianceAsCode/content/applications/openshift/integrity/reject_unsigned_images_by_default/tests/ocp4/e2e-remediation.sh' timed out
```
which looks like a transient failure. The moderate suite didn't run because the cluster didn't install.

**Review by jhrozek — APPROVED** [review]
/lgtm

**Pull request merged**